certified-connectors/microsoftschooldatasyncv2/apiDefinition.swagger.json

```diff
@@ -4,7 +4,7 @@
     "version": "1.0.0",
     "title": "Microsoft School Data Sync V2",
     "description": "Uploads CSV files to School Data Sync Version 2",
-	"contact": {
+    "contact": {
       "name": "School Data Sync Support",
       "url": "https://aka.ms/sdssupport"
     }
```

Comment:
Please add email as well for the contact details, please refer the template 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json
    "contact": {
      "name": "Full name",
      "url": "Contact URL",
      "email": "Contact email address"
    }